client/state/users/reducer.js

```diff
@@ -26,19 +25,6 @@ export function items( state = {}, action ) {
 			return Object.assign( {}, state, {
 				[ action.user.ID ]: action.user,
 			} );
-
-		case USERS_RECEIVE:
```

Comment:
you probably still want this, to be able to receive a set of users at a time?

Reply:
At this time it'll be unused. I think after migrating we would reuse action constants from the current flux users store. I think my [comment below](https://github.com/Automattic/wp-calypso/pull/27900#discussion_r226071613) is related.

Reply:
ah ok. so the only thing using this bit of the tree would be the current user?